helm/botkube/values.yaml

```diff
@@ -150,7 +150,7 @@ config:
     # Settings for Slack
     slack:
       enabled: false
-      channel: 'SLACK_CHANNEL'
+      channel: 'SLACK_CHANNEL'                   # Slack channel name without '#' prefix where you have added BotKube and want to receive notifications in
```

Comment:
Adding doc line above this line will improve readability. Can you please check ?

Reply:
I added it next to variable for the sake of consistency 
https://github.com/infracloudio/botkube/blob/5e3e47f24603b26518c22f2591bc06ee42a392e5/helm/botkube/values.yaml#L151